docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,170 @@
+# Bitbucket integration
+
+!!! note
+    (CUSTOMER FACING MESSAGE NEEDED) We have launched Bitbucket integration to receive your feedback. Some metrics are not available yet.
```

Comment:
I think a message like this could work well! :) I'd only tweak the tone a bit to sound more friendly, for example, "Pulse's Bitbucket integration is new and still in beta, therefore some metrics may not be available just yet. If you'd like to know more about what's coming next, feel free to contact us" <- not sure if here we can use this to open the intercom directly for example 💭 tweak it as you think it's best 🙏 !

Reply:
1. we can link to the Accelerate metrics mentioned they will be the ones supported
2. highlight that historical import won't be available
2. highlight that Deployments will only be detected thought pull requests

"Pulse's Bitbucket integration is new and still in Beta. For now, only the [Accelerate metrics] are available, with Deployments only being detected from Pull requests.
If at first you don't see any data, don't worry. At the moment, we're only collecting new data, but we'll automatically import historical data and backfill your charts within a few weeks."

Reply:
I updated the message based on your feedback. 👍 
I didn't include the "feel free to contact us" link for now. Let me know if you still want to include this link and what is the target url.